curp/src/server/storage/rocksdb.rs

```diff
@@ -81,14 +81,14 @@ impl<C: 'static + Command> StorageApi for RocksDBStorage<C> {
         Ok((voted_for, entries))
     }
 
-    async fn new_snapshot(&self) -> Result<Box<dyn SnapshotApi>, StorageError> {
+    async fn new_snapshot(&self) -> Result<SnapshotUnion, StorageError> {
         // TODO: delete outdated snapshot
         // TODO: better snapshot file naming
```

Comment:
Shall we finish these two todos in this release?

Reply:
this method has some problem, it use Curp storage create xline snapshot, I will refactor this in other PR

Reply:
Ok, please leave an issue to track this problem.

Reply:
#235